sync/models/sync_project_demo.py

```diff
@@ -80,7 +80,7 @@ def odoo_execute_kw(model, method, *args, **kwargs):
         }
 
     @api.model
-    def _eval_context_telegram(self, secrets, eval_context):
+    def _eval_context_telegram_demo(self, secrets, eval_context):
```

Comment:
No need to make changes inside the main module without the need. You can just create unique names inside your module